suite-common/icons/generateIconFont.ts

```diff
@@ -35,6 +35,7 @@ const usedIcons = [
     'bank',
     'bluetooth',
     'bluetoothConnected',
+    'bluetoothSlash',
```

Comment:
Ain't it necessary to increment `runtimeVersion` in `app.config.ts` if you modify icons? 🤔